docs: Adding Kotlin example. (#1454)
aws-powertools/powertools-lambda-java: examples/powertools-examples-core/README.md

> #  Powertools for AWS Lambda - Core Utilities Example

Reviewer: I would keep the Java even if it's kotlin

Reviewer 2: Makes sense, since the library is still native-Java

> #  Powertools for AWS Lambda - Core Utilities Example

Reviewer 2: ```suggestion
#  Powertools for AWS Lambda (Java) - Core Utilities Example
```

Configuration files and deployment process for each tool are described in corresponding README files.

## Kotlin

> For each of the tools, the example application is the same, and consists of the following files:

Reviewer 3: ```suggestion
You can also use Kotlin! The example consists of the following files:
```

Reviewer 3: I think the headings need a rejig - e.g the examples at the top are all java, and then we have the one Kotlin one. 

```md
# Examples
## 1. Java Examples 
gradle, maven, etc.
## 2. Kotlin Examples
```
or something perhaps? It is a bit confusing now and gives me the sense that I could have all the permutations - e.g. CDK + Kotlin.

Reviewer 3: Looks good 👍
